network/src/counters.rs

```diff
@@ -479,7 +479,7 @@ pub static NETWORK_RATE_LIMIT_METRICS: Lazy<HistogramVec> = Lazy::new(|| {
     register_histogram_vec!(
         "diem_network_rate_limit_test",
```

Comment:
Why `_test` in the name?

Reply:
Good question, it shouldn't be there!

I think I had the metrics conflicting with each other on my local testnet.  Guess I've been getting sloppy